src/set_theory/game/pgame.lean

```diff
@@ -255,73 +256,50 @@ instance is_empty_one_right_moves : is_empty (right_moves 1) := pempty.is_empty
 
 /-! ### Pre-game order relations -/
 
-/-- Define simultaneously by mutual induction the `≤` relation and its swapped converse `⧏` on
-  pre-games.
-
-  The ZFC definition says that `x = {xL | xR}` is less or equal to `y = {yL | yR}` if
-  `∀ x₁ ∈ xL, x₁ ⧏ y` and `∀ y₂ ∈ yR, x ⧏ y₂`, where `x ⧏ y` means `¬ y ≤ x`. This is a tricky
-  induction because it only decreases one side at a time, and it also swaps the arguments in the
-  definition of `≤`. The solution is to define `x ≤ y` and `x ⧏ y` simultaneously. -/
-def le_lf : Π (x y : pgame.{u}), Prop × Prop
-| (mk xl xr xL xR) (mk yl yr yL yR) :=
-  -- the orderings of the clauses here are carefully chosen so that
-  --   and.left/or.inl refer to moves by Left, and
-  --   and.right/or.inr refer to moves by Right.
-((∀ i, (le_lf (xL i) ⟨yl, yr, yL, yR⟩).2) ∧ ∀ j, (le_lf ⟨xl, xr, xL, xR⟩ (yR j)).2,
- (∃ i, (le_lf ⟨xl, xr, xL, xR⟩ (yL i)).1) ∨ ∃ j, (le_lf (xR j) ⟨yl, yr, yL, yR⟩).1)
-using_well_founded { dec_tac := pgame_wf_tac }
-
 /-- The less or equal relation on pre-games.
 
 If `0 ≤ x`, then Left can win `x` as the second player. -/
-instance : has_le pgame := ⟨λ x y, (le_lf x y).1⟩
+instance : has_le pgame := ⟨sym2.game_add.fix wf_is_option $ λ x y le,
+  (∀ i, ¬ le y (x.move_left i) (sym2.game_add.snd_fst $ is_option.move_left i)) ∧
+  (∀ j, ¬ le (y.move_right j) x (sym2.game_add.fst_snd $ is_option.move_right j))⟩
 
 /-- The less or fuzzy relation on pre-games.
 
 If `0 ⧏ x`, then Left can win `x` as the first player. -/
-def lf (x y : pgame) : Prop := (le_lf x y).2
+def lf (x y : pgame) : Prop := ¬ y ≤ x
 
 localized "infix (name := pgame.lf) ` ⧏ `:50 := pgame.lf" in pgame
 
-/-- Definition of `x ≤ y` on pre-games built using the constructor. -/
-@[simp] theorem mk_le_mk {xl xr xL xR yl yr yL yR} :
-  mk xl xr xL xR ≤ mk yl yr yL yR ↔
-  (∀ i, xL i ⧏ mk yl yr yL yR) ∧ ∀ j, mk xl xr xL xR ⧏ yR j :=
-show (le_lf _ _).1 ↔ _, by { rw le_lf, refl }
+@[simp] protected theorem not_le {x y : pgame} : ¬ x ≤ y ↔ y ⧏ x := iff.rfl
+@[simp] theorem not_lf {x y : pgame} : ¬ x ⧏ y ↔ y ≤ x := not_not
+theorem _root_.has_le.le.not_gf {x y : pgame} : x ≤ y → ¬ y ⧏ x := not_lf.2
+theorem lf.not_ge {x y : pgame} : x ⧏ y → ¬ y ≤ x := id
 
 /-- Definition of `x ≤ y` on pre-games, in terms of `⧏` -/
```

Comment:
```suggestion
/-- Definition of `x ≤ y` on pre-games, in terms of `⧏`.

The ordering here is chosen so that `and.left` refer to moves by Left, and `and.right` refer to moves by Right. -/
```
with line wrapping. Same below, but with `or.inl` and `or.inr`.